src/components/containers/AnnotationAccordion.js

```diff
@@ -25,51 +16,43 @@ class AnnotationAccordion extends Component {
 
     const key = `annotations[${annotationIndex}]`;
     const value = {text: 'new text'};
-    if (this.context.updateContainer) {
-      this.context.updateContainer({[key]: value});
+
+    if (updateContainer) {
+      updateContainer({[key]: value});
     }
   }
 
   render() {
     const {layout: {annotations = []}} = this.context;
+    const {canAdd, children} = this.props;
+
+    const content =
+      annotations.length &&
+      annotations.map((ann, i) => (
+        <AnnotationFold
+          key={i}
+          annotationIndex={i}
+          name={ann.text}
+          foldIndex={i}
+          canDelete={canAdd}
+        >
+          {children}
+        </AnnotationFold>
+      ));
 
-    const {canAdd, children, localize: _} = this.props;
-
-    const addButton = canAdd && (
-      <Button
-        variant="primary"
-        className="js-add-annotation-button"
-        onClick={this.addAnnotation}
-        icon={<PlusIcon />}
-        label={_('Annotation')}
-      />
-    );
-
-    const panelHeader = canAdd && <PanelHeader action={addButton} />;
-
-    const content = annotations.map((ann, i) => (
-      <AnnotationFold key={i} annotationIndex={i} name={ann.text}>
-        {children}
-      </AnnotationFold>
-    ));
     return (
-      <div className={bem('panel', 'content')}>
-        {panelHeader}
-        {content}
-      </div>
+      <div className={bem('panel', 'content')}>{content ? content : null}</div>
     );
   }
 }
 
 AnnotationAccordion.contextTypes = {
   layout: PropTypes.object,
-  updateContainer: PropTypes.func,
 };
 
 AnnotationAccordion.propTypes = {
-  localize: PropTypes.func,
   children: PropTypes.node,
   canAdd: PropTypes.bool,
 };
 
-export default localize(AnnotationAccordion);
+export default AnnotationAccordion;
```

Comment:
Why are we removing the localization here?

Reply:
because we no longer have any strings to localize in this component :)